### PR TITLE
feat(ui): add auditlog feature detail to auditlog page

### PR DESCRIPTION
**Pull request by kentakozuka**
part of https://github.com/bucketeer-io/bucketeer/issues/804

Untitled

Screenshot 2024-06-25 at 15 00 37

Untitled (9)

### CODE REVIEW

**Comment by cre8ivejp** [comment]
@kentakozuka, I tried to run your branch on my local machine, but it's showing the `SHOW CHANGES` link.

File: ui/web-v2/src/components/AuditLogList/index.tsx [hunk 1]
```diff
@@ -98,6 +104,113 @@ export const AuditLogList: FC<AuditLogListProps> = memo(
...
+  auditLog
+}) => {
+  const [showChanges, setShowChanges] = useState(false);
+  const [showSnapshot, setShowSnapshot] = useState(false);
+  const featureUpdatedEvent = deserializeEvent(auditLog.event!);
+  return (
+    <div>
+      <div className={classNames('text-primary text-xs')}>
+        <button onClick={() => setShowChanges(!showChanges)}>
+          {showChanges ? 'HIDE CHANGES' : '> SHOW CHANGES'}
```
Comment (cre8ivejp) [inline ui/web-v2/src/components/AuditLogList/index.tsx:121]: Please add to the lang assets.

File: ui/web-v2/src/components/AuditLogList/index.tsx [hunk 2]
```diff
@@ -98,6 +104,113 @@ export const AuditLogList: FC<AuditLogListProps> = memo(
...
+            />
+          </div>
+        )}
+      </div>
+      <div>
+        <button
+          onClick={() => setShowSnapshot(!showSnapshot)}
+          className={classNames('text-primary text-xs')}
+        >
+          {showSnapshot ? 'HIDE SNAPSHOT' : '> SHOW SNAPSHOT'}
```
Comment (cre8ivejp) [inline ui/web-v2/src/components/AuditLogList/index.tsx:137]: Same here.

**Comment by kentakozuka** [comment]
@cre8ivejp Fixed and resolved conflicts 👍

**Review by cre8ivejp — APPROVED** [review]
Thank you!

**Pull request merged**